netcore/System.Private.CoreLib/src/System.Threading/Thread.cs

```diff
@@ -181,6 +184,17 @@ public bool Join (int millisecondsTimeout)
 
 		internal void ResetThreadPoolThread ()
 		{
+			if (_name != null) {
+				Name = null;
```

Comment:
These doesn't need braces.

Reply:
Yeah, I can fix that.

On unrelated note, there's plenty of inefficiencies in the `Thread` code in regards to accessing the thread state so I plan to address that in separate PR.